### ISSUE TITLE
Mesa vs netlogo results - predation model

### ISSUE THREAD

**Issue by kleinrap** [projectmesa/mesa#759]
**Describe the bug**
I have been comparing the results from mesa and from netlogo for the the predation model with the same inputs and all parameters equal. I found that the results do not match, models in python (from mesa library) and in netlogo leads to different equilibrium.

I have asked this question in the google group already and was proposed a range of possibilities which I checked. But nothing seemed to fix the issue. I was wondering if anyone here has more insight.

**Expected behavior**

**To Reproduce**
The parameters I am using on both sides are the following:
- Wolf: 25
- Wolf reproduction rate: 5%
- Wolf energy gained from food: 20
- Sheep: 250
- Sheep reproduction rate: 4%
- Sheep energy gained from food: 4
- Grass regrowth time: 30

To have an equivalent grid, I have a 50 by 50 torus grid in mesa.

Note that in NetLogo I changed the code to reflect the mesa movement approach. The movement of the cheap is allowed to be 360 degrees by adding:
rt random 360
lt random 360

**Additional context**

**Comment by jackiekazil** [comment]
To make sure we are looking at the same thing, can you share the following... 
1. link the to netlogo model
2. link to the Mesa model
3. link to the google group conversation

**Comment by kleinrap** [comment]
I have rerun the comparison between both models just now (a year later) and I cannot find the discrepancy I had found before. So it would seem both models do match after all. Apologies for the time lost.
Either way, here is the link to the GitHub with both [model files](https://github.com/kleinrap/predationComparison) and here was the Google group [conversation](https://groups.google.com/forum/#!topic/projectmesa/dsqrIKLJ_64).

**Comment by jackiekazil** [comment]
YAY! That is good news! Thank you for rerunning and providing that insight.

**Comment by lukyfox** [comment]
As I remember, there was an issue with NetLogo probably getting from scheduler (but it could be an older version and maybe it has bee fixed last year...I haven't worked with NetLogo from summer 2019). I noticed that in the NetLogo's wolf-sheep predation model some sheep are some kind of "supersheep"  because they survived the first meeting with wolf. Not all sheep but random number in each simulation. Also when I added hunters and defined a rule, that hunter always kills wolf on the same field there were some wolfs again who survived the first meeting with hunter. Maybe it's just a difference between toy (NetLogo) and tool (MESA) ;) .
